frontend/src/components/common/Cart/CartPopup.jsx

```diff
@@ -9,60 +9,16 @@ import IconButton from '../../ui-kit/components/IconButton';
 
 import styles from './CartPopup.module.scss';
 import { ICONS } from '../../ui-kit/icons';
-
-const items = [
-  {
-    id: 0,
-    title: 'Букет “101 троянда”',
-    img: './images/bouquet.jpg',
-    status: false,
-    oldPrice: 2000,
-    actualPrice: 1500,
-  },
-  {
-    id: 1,
-    title: 'Букет “101 троянда”',
-    img: './images/bouquet.jpg',
-    status: true,
-    oldPrice: 2000,
-    actualPrice: 1500,
-  },
-  {
-    id: 2,
-    title: 'Свій букет #001”',
-    img: './images/bouquet.jpg',
-    status: false,
-    oldPrice: 1550,
-    actualPrice: 1300,
-  },
-  {
-    id: 3,
-    title: 'Букет “101 троянда”',
-    img: './images/bouquet.jpg',
-    status: true,
-    oldPrice: 2000,
-    actualPrice: 1500,
-  },
-  {
-    id: 4,
-    title: 'Свій букет #001”',
-    img: './images/bouquet.jpg',
-    status: true,
-    oldPrice: 1550,
-    actualPrice: 1300,
-  },
-  {
-    id: 5,
-    title: 'Букет “101 троянда”',
-    img: './images/bouquet.jpg',
-    status: false,
-    oldPrice: 2000,
-    actualPrice: 1500,
-  },
-];
+import { calculateTotal } from '../../../redux/slices/cartSlice';
 
 const CartPopup = ({ toggleCart }) => {
-  const totalSum = items.reduce((sum, obj) => sum + obj.actualPrice, 0);
+  const cart = useSelector((state) => state.cartSliceReducer.cartItems);
+  const total = useSelector((state) => state.cartSliceReducer.cartTotalAmount);
```

Comment:
You don't need redux here.

Reply:
Deleted the function calculateTotal from redux. Added a function for calculating total amount to the react component.